### ISSUE TITLE
Problem when use the Directionality widget on rtl

### ISSUE THREAD

**Issue by wahidanvary** [daohoangson/flutter_widget_from_html#1296]
## Steps to Reproduce

<details>
  <summary>HTML</summary>

```<p>براي ايجاد فاصله خالي در متن، كدام كليد مناسب است؟</p>
<p> <kh1>ر1:</kh1>الف) Backspace                   ب)<kh2>ر2:</kh2>Caps Lock                      <kh3>ر3:</kh3>ج) Space Bar                     د)<kh4>ر4:</kh4>Enter</p>
```
</details>

<details>
  <summary>`HtmlWidget` configuration</summary>

```dart
```
</details>

<details>
  <summary>Tesing environment</summary>

```
```
</details>

**Comment by daohoangson** [comment]
To confirm: it works correctly with LTR but doesn't work with RTL?

**Comment by wahidanvary** [comment]
> To confirm: it works correctly with LTR but doesn't work with RTL?
Yes on RTL the position of custom Widgets does not change correctly on every row.

**Comment by wahidanvary** [comment]
> How did you configure the `Directionality` widget? Wrap it outside `HtmlWidget`?

I checked in two ways:
Wrap outside `HtmlWidget`:
`Directionality(
      textDirection: TextDirection.rtl,
      child: HtmlWidget(...`
Or use `dir="rtl"` in the HTML tag.
both of them makes wrong output.

**Comment by wahidanvary** [comment]
I'm sorry. You upgraded to a new version, but the problem hasn't been solved yet. Would you be able to check that?
